utils/api/benefits/entitlementFormula.ts

```diff
@@ -5,7 +5,7 @@ import {
   PartnerBenefitStatusHelper,
 } from '../helpers/fieldClasses'
 import roundToTwo from '../helpers/roundToTwo'
-import { legalValues } from '../scrapers/output'
+import legalValues from '../scrapers/output'
```

Comment:
this line show as error on VS code.  
"... did you mean to use { } ... instead"